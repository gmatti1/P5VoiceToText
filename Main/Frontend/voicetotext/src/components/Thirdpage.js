import React from './../../node_modules/react';
import './../styles/App.css';
import './../../node_modules/bootstrap/dist/css/bootstrap.min.css';
import History from '../containers/History';

function Thirdpage() {
  return (
    <div className='Thirdpage' id='hist'>
       <h1>third page</h1>
      <History />
    </div>
  );
}

export default Thirdpage;
