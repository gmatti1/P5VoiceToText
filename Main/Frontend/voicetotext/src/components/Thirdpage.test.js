import React from 'react';
import {render, cleanup} from '@testing-library/react';
import Thirdpage from './Thirdpage';

 /**
 *
 * This is the test file for Thirdpage component.
 *
 * @version 1.0
 * @author [Shashidhar Reddy Vanteru] <svanter1@asu.edu>
 * @copyright [Copyright 2020, P5VoiceToText]
 * 
 */

afterEach(cleanup)
it('should take a snapshot', () => {
	const { asFragment } = render(<Thirdpage />)
	expect(asFragment(<Thirdpage />)).toMatchSnapshot()
});

