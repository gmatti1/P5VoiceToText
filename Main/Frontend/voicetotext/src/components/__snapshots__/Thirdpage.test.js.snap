// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="Thirdpage"
    id="hist"
  >
    <div
      class="HistoryComponent"
    >
      <form>
        <div
          class="Historytext"
        >
          History
        </div>
        <div
          class="Historylist"
        >
          <select
            class="Historyselect"
            required=""
            size="10"
          />
        </div>
        <div
          class="SearchHist"
        >
          <input
            class="Search"
            placeholder="ï€‚ Search filename.."
            type="text"
            value=""
          />
        </div>
      </form>
      <button
        class="btn btn-danger center"
        id="histcalltopopup"
      >
        See saved result
      </button>
    </div>
  </div>
</DocumentFragment>
`;
